modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -738,23 +738,27 @@ public Table createTable(String name, Consumer<TableChange> tableInitChange) {
             if (tbl != null) {
                 tblFut.completeExceptionally(new TableAlreadyExistsException(name));
             } else {
-                tablesCfg.tables().change(change -> {
-                    if (change.get(name) != null) {
+                tablesCfg.change(tablesChange -> tablesChange.changeTables(tablesListChange -> {
+                    if (tablesListChange.get(name) != null) {
                         throw new TableAlreadyExistsException(name);
                     }
 
-                    change.create(name, (ch) -> {
-                        tableInitChange.accept(ch);
+                    tablesListChange.create(name, (tableChange) -> {
+                        tableChange.changeDataStorage(
+                                dataStorageMgr.defaultTableDataStorageConsumer(tablesChange.defaultDataStorage())
+                        );
 
-                        var extConfCh = ((ExtendedTableChange) ch);
+                        tableInitChange.accept(tableChange);
+
+                        var extConfCh = ((ExtendedTableChange) tableChange);
 
                         tableCreateFuts.put(extConfCh.id(), tblFut);
 
                         // Affinity assignments calculation.
                         extConfCh.changeAssignments(ByteUtils.toBytes(AffinityUtils.calculateAssignments(
                                         baselineMgr.nodes(),
-                                        ch.partitions(),
-                                        ch.replicas())))
+                                        tableChange.partitions(),
```

Comment:
As always, you renamed things for no reason :)

Reply:
It was difficult for me to read what was happening, it became more clear. Should I return?

Reply:
No, you already did the job